var/spack/repos/builtin/packages/py-jupyter-packaging/package.py

```diff
@@ -12,11 +12,15 @@ class PyJupyterPackaging(PythonPackage):
     homepage = "https://github.com/jupyter/jupyter-packaging"
     pypi     = "jupyter_packaging/jupyter_packaging-0.10.4.tar.gz"
 
+    version('0.10.6', sha256='a8a2c90bf2e0cae83be63ccb0b7035032a1589f268cc08b1d479e37ce50fc940')
     version('0.10.4', sha256='589db027cb85a92612f9bcfaeecaa8a9072ac8a4bddaf827f648664258e587c4')
+    version('0.7.12', sha256='b140325771881a7df7b7f2d14997b619063fe75ae756b9025852e4346000bbb8',
+            # name has a '-' instead of a '_'
+            url='https://files.pythonhosted.org/packages/5c/9f/3b332f2a21487026f9521c3fba6efedec77b09f80bffccbb925bba33fcb5/jupyter-packaging-0.7.12.tar.gz')
```

Comment:
```suggestion
            url='https://files.pythonhosted.org/packages/source/j/jupyter_packaging/jupyter-packaging-0.7.12.tar.gz')
```
If you want to make the URL shorter

Reply:
That's better :-) Thanks for the hint!